pep8.py

```diff
@@ -1846,7 +1846,15 @@ def check_files(self, paths=None):
         report.start()
         try:
             for path in paths:
-                if os.path.isdir(path):
+                if path == "-":
+                    # STDIN
+                    path = self.options.stdin_name
```

Comment:
What if someone didn't specify a name? In that case path is `None`. I'm guessing that will break things.